lessons/misc/webdev-intro/lesson.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: 'Intro to web development with HTML/CSS/JS''
+visible: true
+tags:
+  - misc
+  - beginner
+---
+
+- **Authors**: John Ladan [@jladan](https://github.com/jladan)
+- **Research field**: Physics
+- **Lesson topic**: An introduction to web development with HTML/CSS/JS
+- **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/misc/webdev-intro>
+
+# Preface
+
+This is lesson is an introduction to making web applications. A little HTML and CSS are covered as base-level knowledge, but the focus should be on the Document Object Model (DOM) and JavaScript.
+
+The actual lesson contents are in `contents.md`. 
```

Comment:
~~Is it possible to link directly to the contents.md file? Either as a link to the github repo material or (optimally) to the rendered html that (maybe?) shows up on the uoftcoders.github.io website.~~ nevermind, I see that you have the link to your site below.